### ISSUE TITLE
Use statistical data to show strengths and gaps in our coverage

### ISSUE THREAD

**Issue by dommitchell** [DOAJ/Roadmap#33]
Extract statistical data from our index to show the strengths and gaps in our coverage. Find out which areas are underrepresented. 

Check by:
- subject
- language
- country

**Comment by dommitchell** [comment]
This task was held up by a technical block and then deemed to be of lower priority that others. It wasn't completed in its original form but other data from others sources, such as DOAJ's CRM were used instead.

This work will be continued under #69
